app/presenters/schedule_presenter.rb

```diff
@@ -0,0 +1,105 @@
+class SchedulePresenter
+
+  def initialize(venue_object)
+    @venue = venue_object
+  end
+
+  #this is the method actually being called in the view.
+  def shows_at_venue_per_day(week_day)
+    shows_at_venue = Event.where("venue_id" => @venue.id)
+    @shows_at_venue_per_day = []
+    @shows_with_open_time_slots = []
+    shows_at_venue.each do |show|
+      if show.weekday == week_day
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.